src/utils/neynar/exceptions/FetchError.ts

```diff
@@ -0,0 +1,6 @@
+export class FetchError extends Error {
```

Comment:
nit. for next PR, we can probably have Fetch Error shared in a `utils/network` folder internal utils.